lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -716,7 +716,7 @@ namespace Js
             simdFloat32x4TypeDynamic = DynamicType::New(scriptContext, TypeIds_SIMDObject, simdFloat32x4Prototype, nullptr, NullTypeHandler<false>::GetDefaultInstance(), true, true);
 
             simdFloat32x4TypeStatic = StaticType::New(scriptContext, TypeIds_SIMDFloat32x4, simdFloat32x4Prototype , nullptr);
-            simdFloat64x2TypeStatic = StaticType::New(scriptContext, TypeIds_SIMDFloat64x2, simdFloat64x2Prototype, nullptr);
+            //simdFloat64x2TypeStatic = StaticType::New(scriptContext, TypeIds_SIMDFloat64x2, simdFloat64x2Prototype, nullptr);
```

Comment:
I prefer GetScriptContext()->GetConfig(). That means you don't need to disable your tests and pass the extra key to keep it alive. I would settle for `#ifdef SIMD_FLOAT64x2` if you eventually want to get rid of the code. 

---
In reply to: [61961729](https://github.com/Microsoft/ChakraCore/pull/908#discussion_r61961729) [](ancestors = 61961729)